SharePointFramework/PortfolioWebParts/src/data/index.ts

```diff
@@ -440,12 +443,15 @@ export class DataAdapter {
           'GtProjectOwnerId',
           'GtProjectManagerId',
           'GtProjectPhaseText',
+          'GtProjectLifecycleStatus',
+          'GtProjectServiceAreaText',
+          'GtProjectTypeText',
           'GtStartDate',
           'GtEndDate',
           'Title'
         )
         // eslint-disable-next-line quotes
-        .filter("GtProjectLifecycleStatus ne 'Avsluttet'")
+        .filter(getAll ? '' : "GtProjectLifecycleStatus ne 'Avsluttet'")
```

Comment:
Why not have **string** `filter` as a property instead of **boolean** `getAll`?

Reply:
This is because we wanted to expand the possibility to show all projects (including finished ones) when clicking on the "Show all" toggle!

Reply:
I understand that, but it would be better to use `filter` as a parameter instead of `getAll`. Then we support more scenarios then just `"GtProjectLifecycleStatus ne 'Avsluttet'"` or no filter.